hack/gen-crd.sh

```diff
@@ -19,3 +19,10 @@ install_crd \
   "config/crd/bases/metal3.io_provisionings.yaml" \
   "manifests/0000_30_cluster-baremetal-operator_00_metal3provisioning.crd.yaml"
 
+# install bmh crd
+tmpfile="$(mktemp bmhcrd-XXXXXXXX)"
+curl -o $tmpfile https://raw.githubusercontent.com/openshift/baremetal-operator/master/deploy/crds/metal3.io_baremetalhosts_crd.yaml
+if [ $? -eq 0 ]; then
+	install_crd $tmpfile "manifests/0000_30_cluster-baremetal-operator_08_baremetalhost.crd.yaml"
```

Comment:
This CRD can be installed right after the Provisioning CRD, so we can call it 0000_30_cluster-baremetal-operator_01_baremetalhost.crd.yaml

Reply:
Thanks. I've made the change in the latest commit.